config/bundle.Dockerfile

```diff
@@ -3,7 +3,7 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=aws-efs-csi-driver-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=4.11
-LABEL operators.operatorframework.io.bundle.channel.default.v1=4.11
-COPY manifests/4.11/aws-efs-csi-driver-operator.clusterserviceversion.yaml /manifests/aws-efs-csi-driver-operator.clusterserviceversion.yaml
+LABEL operators.operatorframework.io.bundle.channels.v1=preview
+LABEL operators.operatorframework.io.bundle.channel.default.v1=preview
+COPY manifests/stable/aws-efs-csi-driver-operator.clusterserviceversion.yaml /manifests/aws-efs-csi-driver-operator.clusterserviceversion.yaml
```

Comment:
Calling this out in case it raises questions:
I set the labels in this dockerfile to `preview`, same as [LSO](https://github.com/openshift/local-storage-operator/blob/6fbede3ee2f418f5b9977dfc3a3e040d242c0d9a/config/bundle.Dockerfile#L7-L8), because I think this is only used by CI in [openshift/release](https://github.com/openshift/release/blob/26181dde9d758076092009a533a3140b4bf5fcff/ci-operator/config/openshift/aws-efs-csi-driver-operator/openshift-aws-efs-csi-driver-operator-master.yaml#L116) or for dev builds.
AFAIK, the official builds from ART don't need this dockerfile (they rely on config/manifests/art.yaml)